### PR TITLE
Add author details to the Post API response.

### PR DESCRIPTION
**Pull request by andy9775**
Include the Post author information in the response from the get Post(s) api.

### CODE REVIEW

**Review by danielduong8 — APPROVED** [review]
LGTM

File: server/models/post.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export class Post
   UserId!: string; /* Foreign Key from UserId */
 
   static associate(model: any) {
+    Post.belongsTo(model.User);
```
Comment (plausibly) [inline server/models/post.ts:36]: User has 'hasMany' association to Post. Is this fine? I think we should check what the db looks like before merging

Comment (andy9775) [reply server/models/post.ts:36]: This is needed.

**Comment by plausibly** [comment]
LGTM

**Review by plausibly — APPROVED** [review]

**Pull request merged**